client/client.go

```diff
@@ -52,12 +52,14 @@ func (c *UpgradeChecker) Start() {
 }
 
 func (c *UpgradeChecker) run() {
-	ticker := time.NewTicker(c.queryPeriod)
-	defer ticker.Stop()
+	requestInterval := c.InitialRequestInterval
```

Comment:
I am wondering should we make the first call immediately. User might want to know if there is a new version available as soon as they starts the app.

Reply:
Yeah, I made the first call immediately